tallycash/extension
# 💲 Add accounts to all EVM networks

Resolves https://github.com/tallycash/extension/issues/1501

### What
To show and track a new account properly we need to add it to each EVM network we support when the account is added by the user.

### Test

Fresh install
1. Reinstall extension
2. Add wallet with seed
3. Check if the first address is fine on Ethereum mainnet and Polygon (it is displayed on the accounts list)
4. Add some new addresses to the same wallet on Ethereum
5. Switch to Polygon - added addresses should be visible on the list (even without selecting them as active accounts)
6. Add some new addresses to the same wallet on Polygon
7. Switch to Ethereum  - they should be on the list too

Migration testing
1. Checkout `main`
2. Add accounts to Ethereum (2+ addresses), don't switch to Polygon
3. Checkout this branch
4. Switch to Polygon - all the addresses should be there as well

![image](https://user-images.githubusercontent.com/20949277/171860754-a261b85b-fde9-4b5b-af0a-910862ccc555.png)
